src/category_theory/epi_mono.lean

```diff
@@ -40,6 +40,14 @@ begin
     cancel_mono, equiv.apply_eq_iff_eq] at H
 end
 
+instance is_equivalence.epi_map {F : C ⥤ D} [is_equivalence F] {X Y : C} {f : X ⟶ Y} [h : epi f] :
```

Comment:
If this is changed to `is_right_adjoint` then it's more general and implies this instance, right? And dually for below

Reply:
Yup, thanks.